docs/sources/get-started/quick-start.md

```diff
@@ -2,160 +2,277 @@
 
-1. Use Grafana to query the Loki data source.  
+1. Use Grafana to query the Loki data source.
 
-    The test environment includes [Grafana](https://grafana.com/docs/grafana/latest/), which you can use to query and observe the sample logs generated by the flog application.  You can access the Grafana cluster by navigating to [http://localhost:3000](http://localhost:3000).  The Grafana instance provided with this demo has a Loki [datasource](https://grafana.com/docs/grafana/latest/datasources/loki/) already configured.
+   The test environment includes [Grafana](https://grafana.com/docs/grafana/latest/), which you can use to query and observe the sample logs generated by the flog application.
 
-   {{< figure src="/media/docs/loki/grafana-query-builder-v2.png" caption="Grafana Explore" alt="Grafana Explore">}}
+   You can access the Grafana cluster by browsing to [http://localhost:3000](http://localhost:3000).
 
-1. From the Grafana main menu, click the **Explore** icon (1) to launch the Explore tab. To learn more about Explore, refer the [Explore](https://grafana.com/docs/grafana/latest/explore/) documentation.
+   The Grafana instance in this demonstration has a Loki [data source](https://grafana.com/docs/grafana/latest/datasources/loki/) already configured.
 
-1. From the menu in the dashboard header, select the Loki data source (2).  This displays the Loki query editor. In the query editor you use the Loki query language, [LogQL](https://grafana.com/docs/loki/<LOKI_VERSION>/query/), to query your logs.
-    To learn more about the query editor, refer to the [query editor documentation](https://grafana.com/docs/grafana/latest/datasources/loki/query-editor/).
+   {{< figure src="/media/docs/loki/grafana-query-builder-v2.png" caption="Grafana Explore" alt="Grafana Explore" >}}
+
+1. From the Grafana main menu, click the **Explore** icon (1) to open the Explore tab.
+
+   To learn more about Explore, refer to the [Explore](https://grafana.com/docs/grafana/latest/explore/) documentation.
+
+1. From the menu in the dashboard header, select the Loki data source (2).
+
+   This displays the Loki query editor.
+
+   In the query editor you use the Loki query language, [LogQL](https://grafana.com/docs/loki/<LOKI_VERSION>/query/), to query your logs.
+   To learn more about the query editor, refer to the [query editor documentation](https://grafana.com/docs/grafana/latest/datasources/loki/query-editor/).
 
 1. The Loki query editor has two modes (3):
 
    - [Builder mode](https://grafana.com/docs/grafana/latest/datasources/loki/query-editor/#builder-mode), which provides a visual query designer.
    - [Code mode](https://grafana.com/docs/grafana/latest/datasources/loki/query-editor/#code-mode), which provides a feature-rich editor for writing LogQL queries.
 
-   Next we’ll walk through a few simple queries using both the builder and code views.
```

Comment:
I would like to keep this line to let users know that we will be exploring both modes in this quickstart.

Reply:
Restored in cb3506b069

Reply:
Thank you!